packages/foreman/rubygem-rack/rubygem-rack.spec

```diff
@@ -1,28 +1,19 @@
-# template: scl
-%{?scl:%scl_package rubygem-%{gem_name}}
-%{!?scl:%global pkg_name %{name}}
-
+# template: default
 %global gem_name rack
-%global gem_require_name %{gem_name}
 
-Name: %{?scl_prefix}rubygem-%{gem_name}
-Version: 2.2.3.1
+Name: rubygem-%{gem_name}
+Version: 2.2.4
 Release: 1%{?dist}
 Summary: A modular Ruby webserver interface
-Group: Development/Languages
 License: MIT
 URL: https://github.com/rack/rack
 Source0: https://rubygems.org/gems/%{gem_name}-%{version}.gem
 
 # start specfile generated dependencies
-Requires: %{?scl_prefix_ruby}ruby(release)
-Requires: %{?scl_prefix_ruby}ruby >= 2.3.0
-Requires: %{?scl_prefix_ruby}ruby(rubygems)
-BuildRequires: %{?scl_prefix_ruby}ruby(release)
-BuildRequires: %{?scl_prefix_ruby}ruby >= 2.3.0
-BuildRequires: %{?scl_prefix_ruby}rubygems-devel
+Requires: ruby >= 2.3.0
+BuildRequires: ruby >= 2.3.0
+BuildRequires: rubygems-devel
 BuildArch: noarch
-Provides: %{?scl_prefix}rubygem(%{gem_name}) = %{version}
```

Comment:
Why doesn't it provide `rubygem(rack)` anymore? Or does that one get autogenerated on EL8?

Reply:
```console
% rpm -q --provides -p ~/Downloads/rubygem-rack-2.2.4-1.el8.noarch.rpm 
rubygem(rack) = 2.2.4
rubygem-rack = 2.2.4-1.el8
```

seems to be autogenerated, nice